src/app/screens/SignUp/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import wollox from '../../assets/logos/wolox.png';
+
+import Form from './components/Form';
+import styles from './styles.module.scss';
+
+export default function SignUp() {
+  return (
+    <div className={styles.container}>
+      <img src={wollox} alt="wolox" className={styles.image} />
+      <Form />
+      <hr className={`m-top-3 m-bottom-3 ${styles.line}`} />
+      <button type="button" className="button-secondary">
+        Login
+      </button>
```

Comment:
i dont get this button 😄

Reply:
at the moment it does not have functionality, when I make the login screen I add the functionality